src/persistence/surreal/quotes.rs

```diff
@@ -0,0 +1,217 @@
+// ----- standard library imports
+// ----- extra library imports
+use anyhow::{anyhow, Error as AnyError, Result as AnyResult};
+use async_trait::async_trait;
+use cdk::nuts::nut00 as cdk00;
+use surrealdb::Result as SurrealResult;
+use surrealdb::{engine::any::Any, Surreal};
+use uuid::Uuid;
+// ----- local modules
+// ----- local imports
+use crate::credit::quotes;
+use crate::TStamp;
+
+use super::ConnectionConfig;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, strum::Display)]
+enum DBQuoteStatus {
+    Pending,
+    Declined,
+    Accepted,
+}
+impl From<&quotes::QuoteStatus> for DBQuoteStatus {
+    fn from(value: &quotes::QuoteStatus) -> Self {
+        match value {
+            quotes::QuoteStatus::Pending { .. } => Self::Pending,
+            quotes::QuoteStatus::Declined => Self::Declined,
+            quotes::QuoteStatus::Accepted { .. } => Self::Accepted,
+        }
+    }
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+struct DBQuote {
+    quote_id: surrealdb::Uuid, // can't be `id`, reserved world in surreal
```

Comment:
You can use `id`, you just have to be a bit careful about serialization.

You can check out https://github.com/BitcreditProtocol/E-Bill/blob/a2d5bc79f18cc56d7cdabf6055858935afc7a9be/src/persistence/db/notification.rs#L149 how this can be achieved - if you want - quote_id is of course also fine :D